app/assets/javascripts/miq_tree.js

```diff
@@ -73,6 +73,11 @@ function miqOnClickMenuRoles(id) {
   });
 }
 
+function miqTreeSelect(path, key) {
+  var url = path + '?id=' + encodeURIComponent(key.split('__')[0]);
```

Comment:
would it make sense to drop `path`, and just use `'/' + ManageIQ.controller + '/tree_select/?id='`, same as in those `miqOnClick*` functions?